src/test/java/educationalproject/programmingstuff/tests_units/UserControllerTest.java

```diff
@@ -59,17 +62,21 @@ void givenUserRequestWithNoName_whenControllerPassIt_thenUserServiceInvokeApprop
     }
 
     @Test
-    void givenCreateUserPost_whenControllerPassIt_thenUserServiceInvokeAppropriateMethod() {
+    void givenCreateUserPost_whenCreateUser_thenUserServiceInvokeAppropriateMethod() throws Exception {
 
         //Given
-        String testName = null;
 
         //When
-        UserCreateRequestDto userForCreation = TestDataFactory.getUserCreateRequestDtoBuilder()
-                .userName("Sasha")
-                .surname("Grey")
-                .build();
-        userController.createUser(userForCreation);
+        mockMvc.perform(
+                MockMvcRequestBuilders
+                        .post("/users")
+                        .content(jacksonMapper.writeValueAsString(
+                                        UserCreateRequestDto
+                                                .builder()
+                                                .userName("userName")
+                                                .surname("Surname")
+                                                .build()))
+                        .contentType(MediaType.APPLICATION_JSON));
 
         //Then
         Mockito.verify(userService, times(0)).getUsersByName(any());
```

Comment:
this is an overkill, what if you have 100500 methods in your service class? add 100500 verify's to each test?
add this kind of checks only if the tested method really could call some method, but in this case it shouldn't

moreover, there's a convenient method for this, something like verifyNoInteractions

Reply:
Done. Have use "verifyNoMoreInteractions" ("verifyNoInteractions" doesn't work with a methods. Fits only for the whole object. Checked it with documentation. Only the "times(0)" fits for a certain method )